testsuite/setup.py

```diff
@@ -78,8 +78,8 @@ def run(self):
 
 
 # Make sure I have the right Python version.
-if sys.version_info[:2] < (3, 8):
-    print("MDAnalysis requires Python 3.8 or better. "
+if sys.version_info[:2] < (3, 9):
+    print("MDAnalysis requires Python 3.9 or better. "
```

Comment:
```suggestion
    print("MDAnalysis requires Python 3.9 or higher. "
```

Reply:
I like lazy reading, when with 3.9+

Reply:
Yep, that's good too. It's the "better" that I don't like (although 3.11 is supposed to be really better?).